### ISSUE TITLE
Ability to disable minimized player for AndroidTV/FireTV

### ISSUE THREAD

**Issue by k9janer** [TeamNewPipe/NewPipe#4412]
### Checklist

- [X] I checked, but didn't find any duplicates of this issue in the repo. 
- [X] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [X] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

The minimized player isn't really useful on FireTV and it makes navigation very complex using the remote. It would be nice if we could just disable this feature and if it would be disabled by default on FireTV/AndroidTV.

#### Is your feature request related to a problem? Please describe it

For example with version 0.20 if you exit to the menu using the back button while a video is playing, you will end up back in the menu with the video playing in the minimized player but there is no way to close the video on FireTV as you can't move the cursor to the "x" button. Instead you will just scroll down the current video list. For the same reason you can't click on the minimized player to maximize it again. 

If you had a video playing in full screen with version 0.19 all you had to do was press back twice - first close to the video - and the second press to get back to the main menu. Now you have to press back once and then use the arrow keys multiple times on your remote to close the video using the (hard to see) "x" button to land back in the menu without a video playing.

In short: The minimized player makes NewPipe very complex when you don't have a touchscreen or mouse to control the app.

I don't want to discuss multiple issues in one feature request but there are many other minor little issues and hickups navigating the app on FireTV. Would it help if I provide a FireTV Stick to one of the developers to make it easier to test and maintain the App with FireTV/Android TV? Of course this would come without expectations or obligations.

**Comment by avently** [comment]
I don't have a PC near me. What happens you press on back button, and then press back button again? Not double click but two single click. Doesn't the app hides the player and moves focus to trending page?

**Comment by k9janer** [comment]
If you open the app and start playing a video in full screen, you can press back two times to go back to the main page which is then also in focus.

However I have noticed that _the app behaves differently on my Google Pixel 3a (Android 11) and my Fire TV (based on Android 7.1.2)_ here:

Google Pixel 3a: As soon as I touch the back button in full screen mode, I land back on the video info page with the **video paused**. If I press back again, I land on the main page with the video **still paused**.

Fire TV: Pressing the back key while the video is in full screen returns me back to the video info page but the video will **continue to play**. By pressing back again, I return back on the main page with the **video still playing minimized**.

I can now pause the video on FireTV by pressing either ⏸️ or ⏯️ on the remote. However it's not possible to highlight the "X" to close the minimized player.

_Is this different behavior a bug or on purpose?_ The settings on both devices are identical as far as I can see. If the video wouldn't continue to play in the minimized player on FireTV my complaints would still stand but the minimized player would be far less intrusive.

With the current behavior if a video is playing and I want to go back to the main page to choose a new video in silence, I have to either carefully move the cursor to the "x" button at the very top of the currently playing video or I have to press back and then ⏸️/⏯️ to have silence. Reaching the "x" button can take a lot of button presses when you are further down and is also a bit buggy moving trough a long description.

In it's current state the minimized player doesn't benefit FireTV and only makes using the app more complicated:

1) I haven't found a way to "click" on it to go back to reopen the minimized video so it serves no real purpose right now (except for when you want to listen to audio only while browsing your videos maybe).
2) It requires additional and very deliberate steps to avoid having the video continue playing in the background.
3) It takes up some space on the screen for a feature you can't really use.
4) It displays an "x" to close button that you can't interact with (unless you connect a touch pad or mouse).
5) The minimized player is even broken: It doesn't continue to play the video by pressing the ⏯️ or ▶️ button on the remote anymore after you've returned to the main page after watching 2+ videos in a row. (One of the many minor bugs using a keyboard/remote to control the app I've mentioned)

**Comment by avently** [comment]
Different behavior for phones and tablets/TVs is made like this (paused/ not paused), it's expected. 

Issues you mentioned for Android TV is something that should be improved by someone who uses Android TV and knows what behavior to choose in all the cases. Everything you mentioned can be changed easily. So just wait when somebody with TV will be interested in it. I also think that mini player isn't useful for TVs right now, I leave it in a working state so somebody could decide what to do with it. 

Video can be stopped once you get a mini player collapsed (like after second back press) - it's a couple of lines of new code.

I don't know how you could interact with background player before (in 0.19.8) but now there is a way for it - it's mini player. Yes, not right now, because it's inaccessible but it could be accessible when someone will find a way to move focus in a correct order

**Comment by k9janer** [comment]
Okay, thank you for looking into this.

A more simplified way to control and navigate the app would be welcome.

I'm still open to send my 2nd gen FireTV Stick to a developer. I would also add a Y usb cable so you can connect a PC or usb stick. This would come without expectations or obligations of course.

**Comment by SameenAhnaf** [comment]
@mhmdanas This should be closed in favor of #5590.
